src/Expectation.php

```diff
@@ -154,9 +154,10 @@ public function sequence(...$callbacks): Expectation
             throw new BadMethodCallException('Expectation value is not iterable.');
         }
 
-        $value  = is_array($this->value) ? $this->value : iterator_to_array($this->value);
-        $keys   = array_keys($value);
-        $values = array_values($value);
+        $value          = is_array($this->value) ? $this->value : iterator_to_array($this->value);
+        $keys           = array_keys($value);
+        $values         = array_values($value);
+        $callbacksCount = count($callbacks);
```

Comment:
I don't really see the need for the temporary variable, especially as we don't use it later. Might as well just inline it further down

Reply:
The count of `$callbacks` may increase in this line. So I used temporary variable.

https://github.com/pestphp/pest/blob/a16a19e1214f4e9e817b775ab395dcb52e45c42b/src/Expectation.php#L165

The count is increasing for the following feature.

https://github.com/pestphp/pest/blob/2f0cd7a4e3a36ac51367ea45d65fc09403ec32c7/tests/Features/Expect/sequence.php#L18-L27

Reply:
My bad. Doh! It's getting late over here 😁

Reply:
No one saw it, don't worry. hahah